extensions/desktop/common/fonts

```diff
@@ -25,6 +25,13 @@ else
   ARCH="$SNAP_ARCH-linux-gnu"
 fi
 
+if ! snapctl is-connected "%PLATFORM_PLUG%"; then
```

Comment:
I talked to the snapd folks and it seems that this might just be a snapd bug proper, this is still valida workaround, but it seems that it still makes sense to create fonts.conf below so that once chromium gets connected the FONCCONFIG_FILE exist and treated appropriately at first launch.

So maybe, to be future proof, just gate this conditional to running fc-cache. I also don't see this being an issue per se anymore considering the `|| echo below`, but the check is not harmful.

Reply:
Right, given the `|| echo` below, this check isn't necessary anymore in the configure hook.
I'll remove it. It is still valid in desktop-exports I think.